engine/lib/admin.php

```diff
@@ -598,6 +598,8 @@ function _elgg_admin_plugin_screenshot_page_handler($pages) {
 function _elgg_admin_markdown_page_handler($pages) {
 	set_input('plugin_id', elgg_extract(0, $pages));
 	set_input('filename', elgg_extract(1, $pages));
+	
+	elgg_set_context('admin');
```

Comment:
not doing anything now about it, as this is the same behaviour as before the resources views implementation